core/src/test/java/org/frankframework/http/cxf/SoapProviderTest.java

```diff
@@ -371,12 +371,8 @@ public Message processRequest(Message message, PipeLineSession pipelineSession)
 				try {
 					MatchUtils.assertXmlEquals(getFile("VrijeBerichten_PipelineRequest.xml").asString(), message.asString());
 
-					// Ensure the message is registered on the PipelineSession.
-					pipelineResult.closeOnCloseOf(pipelineSession);
```

Comment:
This test verifies that after the pipeline has processed the message (and used to be closed) it would still be able to read the response. Now that `scheduleCloseOnSessionExit` and `closeOnCloseOf` no longer exist I sincerely doubt that this will be an issue.

I don't think we should remove the test(s), as they are still useful, but the names might be a bit confusing.

Reply:
I'll check these tests, if I can do something to improve what they actually want to test.